frontend-cert/js-projects/lab-factorial/user-stories.md

```diff
@@ -0,0 +1,8 @@
+1. You should ask the user for a number using a prompt window and store the number in a variable called `inputNumber`.
+   
+2. Declare a variable `result` and assign it the value of 1.
+   
+3. Using a for loop, you should loop through all numbers from `1` to `inputNumber` (inclusive) and for each number, multiply the `result` variable by the current number.
+   
+4. You should log the result of the factorial of `inputNumber` to the console. The output should be in the format `The factorial of [inputNumber] is [result]`.
```

Comment:
```suggestion
4. You should log the result of the factorial of `inputNumber` to the console. The output should be in the format `The factorial of <inputNumber> is <result>`.
```
I'm not sure if this way is better, but when we ask to follow a certain format, I'd like to keep the same placeholder format everywhere, for consistency.

Reply:
By the way, I'm okay with changing the projects in which we use `<>` into `[]`. It works either way for me.

Reply:
We can decide one format for consistency.

Reply:
I think I've typically used the <>, but I think the brackets might be better. If the <> aren't inside a code block, I think they will be interpreted as HTML if we are rendering the markdown.